waku/v2/node/storage/message/message_store.nim

```diff
@@ -8,12 +8,13 @@ import
 ## trieve historical messages
```

Comment:
Retrieve? 😛

Reply:
Right! :D

Reply:
BTW, the missing `re` was in the line above!

Reply:
Glad you find the missing letters!